meta-iot2000-example/README.SWUPDATE.md

```diff
@@ -0,0 +1,63 @@
+# Example for efibootguard + SWUpdate support with meta-iot2000
+
+[SWUpdate](https://github.com/sbabic/swupdate) is a software that supports
+firmware and/or software updates targeting embedded devices.
+
+It can be run as a stand-alone server in foreground or as a daemon listening
+for connections via ethernet. Different backends can be configured to support
+other 3rd party software to provide the update artifacts. These are then pulled
+by SWUpdate to update the system.
+
+The new boot loader EFI Boot Guard will take care that unsuccessful firmware
+updates will be rolled back so that the system is not bricked.
+
+To build the image, the kas tool is utilized.
+
+The [kas](https://pypi.python.org/pypi/kas) recipes in
+
+```
+meta-iot2000/meta-iot2000-example/kas-update{,-rt}.yml
+```
+
+build the base image as well as an update artifact without and with a
+real-time kernel respectively, ready to be fed to `SWUpdate`. The recipes
+contain all needed layer references.
+
+For SWUpdate integration, the official
+[meta-swupdate](https://github.com/sbabic/meta-swupdate) layer is used. Per
+default it installs an init script to start up a webserver for update purposes,
+which is not needed for manual updates. For this purpose, it is replaced by an
+empty init script.
+
+This example of `SWUpdate` and `efibootguard` integration provides manual
+update capability only.
+
+
+The manual update process means, that the `swu` file must be copied onto the
+finished SD-card. To use it with swupdate, the following command is used:
```

Comment:
.. or via SSH or something to the booted device

Reply:
Added